plugins/ulimit-adjuster/adjuster.go

```diff
@@ -23,7 +23,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/containerd/containerd/log"
+	"github.com/containerd/log"
```

Comment:
wondering if this plugin should also be its own module

Reply:
Possibly.  The other plugins are, but it wasn't clear to me about a specific background.  Would it help for removing dependencies?

Reply:
Sorry, I failed to spot the plugin not being a module of its own in the original review. Yes, the background is to prevent any plugin-specific dependencies to sneak into NRI's deps. In principle it would be enough to only do this if the plugin had some new plugin-specific deps. But instead we usually just blindly make each plugin a module of its own.

Reply:
👆 this was done in https://github.com/containerd/nri/pull/54